web_src/js/features/repo-diff.ts

```diff
@@ -234,17 +232,80 @@ function initRepoDiffShowMore() {
   });
 }
 
+function shouldStopLoading() {
+  const hash = window.location.hash;
+
+  if (!hash) {
+    return true; // No hash to look for
+  }
+
+  // eslint-disable-next-line unicorn/prefer-query-selector
+  const targetElement = document.getElementById(hash.substring(1));
```

Comment:
```suggestion
  if (!window.location.hash) {
    return true; // No hash to look for
  }

  // eslint-disable-next-line unicorn/prefer-query-selector
  const targetElement = document.getElementById(window.location.hash.substring(1));
```

No need for that variable imho.